pages/reflections.md

```diff
@@ -1,11 +1,47 @@
+[Home](https://michaelrappa.github.io/usa-migration/) |
+[Analysis](https://michaelrappa.github.io/usa-migration/pages/analysis.html) |
+[Background](https://michaelrappa.github.io/usa-migration/pages/background.html) |
+[Reflections](https://michaelrappa.github.io/usa-migration/pages/reflections.html) |
+[Workflow](https://michaelrappa.github.io/usa-migration/pages/workflow.html) |
+[Credits](https://michaelrappa.github.io/usa-migration/pages/credits.html) |
+[Bibliography](https://michaelrappa.github.io/usa-migration/pages/bibliography.html) |
+[Sources](https://michaelrappa.github.io/usa-migration/pages/sources.html)
+
+
+
 # Reflections
 
-This is sort of your final thoughts, a summary of the process of the project itself. By discussing where you hit roadblocks and where you wish you could go if you had more time, you leave future individuals something to build on.
+Our project was overall a success, however due to time and skill restraints, we unfortunately were not able to push this topic to the furthest possible conclusions. Had these factors not been in place, the following is what we would have done to supplement our work:
+* Gathering more data/ Compiling multiple data sources
+* Delving deeper into historical and political literature surrounding critical bills and events affecting immigration.
+* More diverse modes of visualization
+
+---
+
+* _Gathering more data:_ 
+
+   The simple fact of a project like this is that we were at the mercy of the data. Essentially, with our limited time and developing skills, we could not undertake a huge data gathering endeavor, meaning that our scope was restricted to the data that was already out there. Had those limitations not been in place, we would have spent more time digging into the data to get the most complete figures possible. This ties into compiling multiple data sources. The dataset we primarily used had data from each decade rather than each year. That data resolution was sufficient for the scope of our project, but for more intimate conclusions to be drawn, more intricate and complete data would be required. This would mean doing something wholly out of our wheelhouse by taking multiple datasets and compiling them to fill each other’s gaps. Had our own skills in this arena not been a factor, compiling datasets would be a necessary next step for deeper conclusions to be worked towards. This, of all the things that could be done, would be the most useful and necessary step moving forward with this, not just for this project, but for posterity on the subject as well.
```

Comment:
Good overall summations of each area of work, I like the breakdown! Furthermore, you don't give it a short shrift, as some projects have in the past.